package.json

```diff
@@ -63,8 +63,8 @@
 		"lint:code": "xo --ignore test/external/",
 		"lint:rest": "prettier --ignore-path .gitignore --ignore-unknown --check .",
 		"lint": "run-s lint:*",
-		"autofix:code": "run-s lint:code --fix",
-		"autofix:rest": "run-s lint:rest --write .",
+		"autofix:code": "npm run lint:code -- --fix",
+		"autofix:rest": "npm run lint:rest -- --write .",
```

Comment:
I know, I just bitched about including unrelated stuff in a PR, and then I went and did the same thing 🤦

Reply:
Lmaooo happens XD